src/App.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
-   
+import { Header } from './components/header/Header';
```

Comment:
Change the folder name to "Header" and change the import to './components/Header'

Reply:
Done